ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -122,6 +122,16 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.neighbors[neighbor].route_map_in is arista.avd.defined %}
    neighbor {{ neighbor }} route-map {{ router_bgp.neighbors[neighbor].route_map_in }} in
 {%         endif %}
+{%         if router_bgp.neighbors[neighbor].default_originate is arista.avd.defined %}
+{%             set default_originate_cli = "neighbor " ~ peer_group ~ " default-originate" %}
```

Comment:
replace ```peer_group``` with ```neighbor```

Reply:
If you are looking to generate similar config for a PEER_GROUP, Place your code in the lines between 22 to 77.